### PR TITLE
Fix Yamler issue and IP geolocation (#42 & #44)

### PR DESCRIPTION
**Pull request by qlimax5000**

### CODE REVIEW

File: src/main/java/fr/Alphart/BAT/Configuration.java [hunk 1]
```diff
@@ -45,12 +45,18 @@ public Configuration(){
 	
 	@Comment("Set to true to use MySQL. Otherwise SQL Lite will be used")
 	@Setter
+    @Path(value = "mysql.enabled")
 	private boolean mysql_enabled = true;
+    @Path(value = "mysql.user")
 	private String mysql_user = "user";
+    @Path(value = "mysql.password")
 	private String mysql_password = "password";
+    @Path(value = "mysql.databse")
```
Comment (mikroskeem) [inline src/main/java/fr/Alphart/BAT/Configuration.java]: `mysql.databse` - nice typo, this confused me a lot

**Comment by qlimax5000** [comment]
@mikroskeem Whoops! My bad... I wrote it pretty late because I wanted a fix quickly. I've committed a fix. Nice catch, thanks!

**Comment by mikroskeem** [comment]
@alphartdev bump?

**Comment by alphartdev** [comment]
The Yamler fix is working fine... Testing the new IPgeolocation vendor

**Comment by alphartdev** [comment]
Working fine as well!
Great job merging all that together

**Pull request merged**